go-controller/pkg/node/gateway_iptables.go

```diff
@@ -162,6 +178,45 @@ func getNodePortIPTRules(svcPort kapi.ServicePort, targetIP string, targetPort i
 	}
 }
 
+func getClusterIPITPLocalIPTRules(svcPort kapi.ServicePort, clusterIP string, svcHasLocalHostNetEndPnt bool) []iptRule {
```

Comment:
you could just make this function name: getITPLocalIPTRules since you already mentioned in the docs it applies to cluster ip only